client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -351,3 +352,25 @@ function useShowSiteCreationNotice( allSites: SiteExcerptData[], newSiteID: numb
 		window.history.replaceState( null, '', newUrl.toString() );
 	}, [ __, allSites, dispatch, newSiteID ] );
 }
+
+function useShowSiteTransferredNotice() {
+	const { __ } = useI18n();
+	const dispatch = useDispatch();
+	const shownSiteTransferredNotice = useRef( false );
+
+	useEffect( () => {
+		if ( shownSiteTransferredNotice.current ) {
+			return;
+		}
+		const url = new URL( window.location.href );
+		if ( url.searchParams.get( 'transfer_complete' ) === 'true' ) {
```

Comment:
Can we use `site-transfer-complete` for consistency with `site-transfer-confirm`?

Reply:
I added it like that, as originally, it was `&msg=transfercomplete`. I wanted to kinda keep the same identifier. Do you think it makes more sense? If not, I can change it to `site-transfer-complete`